spec/index.html

```diff
@@ -492,9 +492,13 @@ <h3>IRIs</h3>
       of [[!RFC3987]]. Further normalization MUST NOT be performed when
       comparing IRIs for equality.</p>
 
-      <p class="issue" data-number="4">
-        [[RFC3987]] never made it past the Proposed Standard phase
-        and should not be normatively cited.
+      <p class="issue" data-number="15">
```

Comment:
This isn't a fair balance of the situation. 

Simply state that the WG is aware RFC3987 has not progressed (avoid "abandoned").

In general, I don't think these notes should present only one choice nor should assume the first suggestion is the direction of the working group.